storage/etcd/config.go

```diff
@@ -0,0 +1,82 @@
+package etcd
+
+import (
+	"time"
+
+	"github.com/coreos/dex/storage"
+	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/pkg/transport"
+	"github.com/sirupsen/logrus"
+)
+
+type EtcdSSL struct {
+	ServerName string `json:"server_name" yaml:"server_name"`
```

Comment:
nit: we use camel case for struct tags. `json:"serverName"`

Reply:
Why don't we just use `transport.TLSInfo` https://github.com/coreos/etcd/blob/master/pkg/transport/listener.go#L59?

```go
tlsInfo := transport.TLSInfo{
    CertFile:      "/tmp/test-certs/test-name-1.pem",
    KeyFile:       "/tmp/test-certs/test-name-1-key.pem",
    TrustedCAFile: "/tmp/test-certs/trusted-ca.pem",
}
tlsConfig, err := tlsInfo.ClientConfig()
...
clientv3.New(clientv3.Config{
    ...
    TLS:         tlsConfig,
})
```

Reply:
Ok, nvm. It's for json, yaml tagging.